sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -2140,44 +2155,67 @@ void ProgramManager::removeImages(sycl_device_binaries DeviceBinary) {
     // Unmap the unique kernel IDs for the offload entries
     for (sycl_offload_entry EntriesIt = EntriesB; EntriesIt != EntriesE;
          EntriesIt = EntriesIt->Increment()) {
-
+      const char *Name = EntriesIt->GetName();
```

Comment:
Can it be `std::string_view` instead?

Reply:
I changed it to `KernelNameStrT`, which maps to `std::string_view` in preview builds and `std::string` otherwise, since there are a lot of containers involved that use this key type and don't have heterogenous lookup support.